include/LumicalHit.hh

```diff
@@ -1,39 +1,30 @@
 #ifndef LumicalHit_h
 #define LumicalHit_h 1
- 
+
 #include "G4VHit.hh"
 #include "G4THitsCollection.hh"
 #include "G4Allocator.hh"
 #include "G4ThreeVector.hh"
 #include "G4LogicalVolume.hh"
 #include "G4Transform3D.hh"
-#include "G4RotationMatrix.hh" 
+#include "G4RotationMatrix.hh"
 #include "G4Types.hh"
 
 class LumicalHit : public G4VHit
 {
    public:
       LumicalHit(G4int pl, G4int zn, G4int pd);
-      LumicalHit(const LumicalHit& right);
       virtual ~LumicalHit();
-            
-      const LumicalHit& operator=(const LumicalHit& right);
-      G4bool operator==(const LumicalHit&) const;
+
 
       inline void *operator new(size_t);
       inline void  operator delete(void* aHit);
-      
-      G4int plan1, zone, pad;
-      G4double totene;
-      G4double x, y, z, px, py, pz;
```

Comment:
If we use fVar convention for internal class variables, lets use it consistently for all variables to not get confused